Update README.md (#1330)
objectiv/objectiv-analytics: README.md

</a>

[Objectiv](https://objectiv.io/) is an open-source data collection & modeling platform that helps data teams run product analytics from their notebooks, with full control over data and models.

![image](https://user-images.githubusercontent.com/82152911/194335360-fa763b8f-eea5-4d70-819c-dd0602ab8f47.png)

* **Control product analytics in code** - Take product analytics into your own hands with full access to data and models from your Python notebook.
* **Explore & model with zero grunt work** - Work directly on super-structured raw data, with pre-built models designed for product analytics.
* **First-party data & built for integration** - Objectiv connects to your own data store of choice and plays nice with most tools in the modern data stack.
* **Set up error-free tracking with strict validation** - Get helpful tooling to test, validate and debug your tracking setup at multiple stages. No more surprises downstream.

[Official Documentation](https://objectiv.io/docs)

---

## Get started

### 1. Check out the example notebooks

> A number of [example notebooks](/modeling/example-notebooks/index.mdx) are available to demonstrate what you can do with Objectiv. 

Reviewer: ```suggestion
A number of [example notebooks](https://objectiv.io/docs/modeling/example-notebooks/) are available to demonstrate what you can do with Objectiv. 
```

### 2. Get Objectiv Up

> [Objectiv Up](https://objectiv.io/docs/home/up.md) is a pre-packaged, dockerized version of Objectiv's powerful open-source analytics platform that can be self-hosted in under five minutes.

Reviewer: ```suggestion
[Objectiv Up](https://objectiv.io/docs/home/up/) is a pre-packaged, dockerized version of Objectiv's powerful open-source analytics platform that can be self-hosted in under five minutes.
```

All the necessary infrastructure comes ready to go. Pre-built notebooks, models & dashboards are included for common product & marketing use cases, so you can get to work directly. A demo app & data are included for testing.

![objectiv-up-included](https://user-images.githubusercontent.com/82152911/194336454-e51de234-0975-442b-8d1a-8971657bc27f.svg)

> As a result of [strictly structured data collection](https://objectiv.io/docs/taxonomy), you can simply replace the demo app with your own app when you’re ready, and reuse all the notebooks and dashboards on your own data. They have been battle-tested and are ready to be used in production.

#### ... or go fully managed with Objectiv Cloud

> Objectiv is also available as [a fully managed cloud setup](https://objectiv.io/docshttps://www.objectiv.io/cloud), enabling you to run Objectiv's open-source analytics platform at scale without worrying about the Ops part. It connects to your own data store, so you'll remain in full control over your data.

Reviewer: ```suggestion
Objectiv is also available as [a fully managed cloud setup](https://www.objectiv.io/cloud), enabling you to run Objectiv's open-source analytics platform at scale without worrying about the Ops part. It connects to your own data store, so you'll remain in full control over your data.
```
